ls1intum/Artemis
# remove slide-toggle and combine GI and SGI editors

### Checklist

### Motivation and Context


resolve #1396
### Description

Some users found the toggle to be confusing. Since an exercise can have both free text grading instructions and structured grading instructions we decided to add the free text grading instructions in the same editor as the structured grading instructions. When parsing the editor content we interpret the content as free text grading instruction if it does not start with a known command identifier of a structured grading instruction
### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. create or edit Text/Modeling/ file upload exercise
- add GI and SGI in the same editor and make sure they are displayed in the exercise details view as well as in the tutor assessment view. Make sure SGI are still draggable and all functionality is fulfilled
- add GI only  and make sure they are displayed in the exercise details view as well as in the tutor assessment view.
- add SGI only  and make sure they are displayed in the exercise details view as well as in the tutor assessment view.
all in the same editor

### Screenshots


![image](https://user-images.githubusercontent.com/50635005/82132954-03307100-97e6-11ea-8b88-3ca2fe4df6e1.png)
